apps/site-nft/src/components/App/index.tsx

```diff
@@ -24,7 +24,7 @@ import styles from './styles.module.less';
 
 
 export const App = () => {
-
```

Comment:
So, yes, sometimes we still gotta do `printf` debugging (this was part of trying to figure out why our environment wasn't getting mixed into the code properly)